### DISCUSSION TITLE
dropdown icon not showing on initial load, but showed when refresh

### DISCUSSION THREAD

**Discussion by fathurqarnain** [quasarframework/quasar#14754] [General - Components / Directives / etc]
hi, I found a problem with the quasar dropdown button, the icon on the right of the dropdown doesn't exist during initial load, but when I reload it is there, has anyone experienced something similar?

Here is the code :
![Screenshot from 2022-10-31 13-08-26](https://user-images.githubusercontent.com/103421451/198943331-415820ca-3c60-4c7b-8d3f-f7b9efbaab2e.png)

here is when initial load : 
![Screenshot from 2022-10-31 13-10-54](https://user-images.githubusercontent.com/103421451/198943390-457ecfc9-7829-4657-9a39-abf063280283.png)

after refreshed :
![Screenshot from 2022-10-31 13-11-07](https://user-images.githubusercontent.com/103421451/198943431-53650b3d-64ce-4f25-a2f9-64df39c03a05.png)

did somebody know the cause of this problem?
thank you

**Comment by MilosPaunovic** [comment]
Create a Codepen reproduction.
